doc/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@libp2p/docs",
+  "version": "1.0.0",
+  "description": "Docs for libp2p",
+  "license": "Apache-2.0 OR MIT",
+  "homepage": "https://github.com/libp2p/js-libp2p/tree/master/packages/interface-internal#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/libp2p/js-libp2p.git"
+  },
+  "bugs": {
+    "url": "https://github.com/libp2p/js-libp2p/issues"
+  },
+  "type": "module",
+  "eslintConfig": {
+    "extends": "ipfs",
+    "parserOptions": {
+      "sourceType": "module"
+    }
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "import": "./dist/src/index.js"
+    }
+  },
+  "scripts": {
+    "clean": "aegir clean",
+    "lint": "aegir lint",
+    "build": "aegir build",
+    "doc-check": "aegir doc-check"
+  },
+  "dependencies": {
+    "@chainsafe/libp2p-yamux": "^4.0.2",
+    "@libp2p/interface-metrics": "^4.0.8",
+    "@libp2p/mplex": "^8.0.4",
+    "@libp2p/prometheus-metrics": "^1.1.5",
+    "@libp2p/tcp": "^7.0.3",
+    "aegir": "^39.0.13",
+    "libp2p": "^0.45.9",
+    "prom-client": "^14.2.0"
+  }
```

We don't want to publish this to npm

```suggestion
  },
  "private": true
```